src/hooks/useSetSelectedEntity.tsx

```diff
@@ -4,17 +4,21 @@ import { useState, useEffect, SetStateAction, Dispatch } from 'react';
 export const useSetSelectedEntity = (
   entityValue: string,
   cookieName: string,
-): { selectedEntity: string; setSelectedEntity: Dispatch<SetStateAction<string>> } => {
+): {
```

Comment:
let's move this return type of the hook to src/types folder, it looks ugly when long types are defined directly in functions or classes